package/CHANGELOG

```diff
@@ -39,6 +39,8 @@ Fixes
   * fixes guessed_attributes and read_attributes methods of the Topology class, as 
     those methods were giving unexpected results for some topology attributes
     (e.g. bonds, angles) (PR #3779).
+  * Add pickling support for Atom, Residue, Segment, ResidueGroup 
```

Comment:
Entries newest first and this needs to be put under 2.5.0. Merging with upstream/develop also required. :)